<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogCabinetPostion</class>
 <widget class="QDialog" name="DialogCabinetPostion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>773</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>机柜位置</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#DialogCabinetPostion{background-image: url(:/image/box_background);}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,1">
   <item>
    <widget class="WidgetPduDevice" name="widget_left" native="true">
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>机柜</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="WidgetCabinetPostion" name="tableWidget_cabinet">
       <property name="maximumSize">
        <size>
         <width>180</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QTableWidget{
	border-left:5px solid rgb(12,62,180);
	border-right:5px solid rgb(12,62,180);
	border-top:8px solid rgb(12,62,180);
	border-bottom:20px solid rgb(12,62,180);
	gridline-color: rgb(12,62,180);
}
QTableWidget::item
{
/*	border: 1px solid rgb(102,198,249);*/
	color: gray;
}
/*QTableWidget::item:selected
{
	background: rgb(44,105,182);
	color: white;
}
QTableWidget::item:hover:!selected
{
	background: rgb(102,198,249);
	color: white;
}*/</string>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustToContents</enum>
       </property>
       <property name="autoScroll">
        <bool>true</bool>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="showDropIndicator" stdset="0">
        <bool>false</bool>
       </property>
       <property name="dragDropOverwriteMode">
        <bool>false</bool>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::NoSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="textElideMode">
        <enum>Qt::ElideLeft</enum>
       </property>
       <property name="showGrid">
        <bool>true</bool>
       </property>
       <property name="gridStyle">
        <enum>Qt::SolidLine</enum>
       </property>
       <property name="sortingEnabled">
        <bool>false</bool>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="cornerButtonEnabled">
        <bool>false</bool>
       </property>
       <property name="rowCount">
        <number>0</number>
       </property>
       <property name="columnCount">
        <number>1</number>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderHighlightSections">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>25</number>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderDefaultSectionSize">
        <number>9</number>
       </attribute>
       <attribute name="verticalHeaderHighlightSections">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderMinimumSectionSize">
        <number>4</number>
       </attribute>
       <attribute name="verticalHeaderStretchLastSection">
        <bool>false</bool>
       </attribute>
       <column/>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="WidgetPduDevice" name="widget_right" native="true">
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
   </item>
  </layout>
  <action name="action_add">
   <property name="text">
    <string>添加机架设备</string>
   </property>
  </action>
  <action name="action_delete">
   <property name="text">
    <string>删除机架设备</string>
   </property>
  </action>
  <action name="action_modify">
   <property name="text">
    <string>修改机架设备</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WidgetPduDevice</class>
   <extends>QWidget</extends>
   <header>widgetpdudevice.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetCabinetPostion</class>
   <extends>QTableWidget</extends>
   <header>widgetcabinetpostion.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
